<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>324</width>
    <height>203</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>60</y>
      <width>131</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>name: (s0000) </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>68</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>end:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="newShotCam_B">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>140</y>
      <width>131</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>newShot/Cam</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="start_LE">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>30</y>
      <width>61</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="shots_LV">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>161</width>
      <height>171</height>
     </rect>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::NoSelection</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>68</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>start:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="name_LE">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>80</y>
      <width>131</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="newShot_B">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>110</y>
      <width>131</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>newShot</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>101</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>currentShots:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="end_LE">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>30</y>
      <width>61</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>start_LE</tabstop>
  <tabstop>end_LE</tabstop>
  <tabstop>name_LE</tabstop>
  <tabstop>newShot_B</tabstop>
  <tabstop>newShotCam_B</tabstop>
  <tabstop>shots_LV</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
